go.mod

```diff
@@ -6,12 +6,12 @@ require (
 	github.com/go-logr/logr v1.2.3
 	github.com/onsi/ginkgo/v2 v2.2.0
 	github.com/onsi/gomega v1.20.2
-	github.com/projectcontour/contour v1.22.1
+	github.com/projectcontour/contour v1.23.0
 	github.com/spf13/cobra v1.5.0
 	github.com/spf13/viper v1.13.0
-	k8s.io/api v0.24.6
-	k8s.io/apimachinery v0.24.6
-	k8s.io/client-go v0.24.6
+	k8s.io/api v0.25.2
```

Comment:
`ENVTEST_K8S_VERSION` is still 1.24 while `go.mod` dependencies are updated to 1.25.
Please use the consistent k8s version. I think either version (1.24 or 1.25) would be ok.

It is helpful to check no previous versions are remaining like: `git grep -Fw 24`.

Reply:
Thank you for your review.

I forgot checking it and fixed it.

These k8s related packages such as`k8s.io/api` are depend on the contour version.
Therefore I updated `ENVTEST_K8S_VERSION` to `v1.25.0`.